adapter/internal/ga/client.go

```diff
@@ -288,13 +290,98 @@ func addAPIToChannel(resp *discovery.DiscoveryResponse) {
 			IsDeployEvent:    false,
 			OrganizationUUID: gaAPI.OrganizationUUID,
 			RevisionUUID:     gaAPI.RevisionUUID,
+			DeployedEnv:      gaAPI.DeployedEnv,
 		}
 		GAAPIChannel <- event
 		delete(apiRevisionMap, apiEntry)
 		logger.LoggerGA.Infof("API Undeploy event is added to the channel. : %s", apiEntry)
 	}
 }
 
+func addAPIWithEnvToChannel(resp *discovery.DiscoveryResponse) {
+
+	// removedAPIEnvMap is used To keep track of the APIs needs to be deleted.
+	var removedAPIEnvMap = make(map[string]map[string]*ga_model.Api)
+
+	if !isFirstResponse {
+		for k, v := range apiEnvRevisionMap {
```

Comment:
shall we have meaningful terms for single character variable names.

Reply:
fixed via https://github.com/wso2/product-microgateway/commit/ae2c06882353bda7a59101381ef3e82f88667a21